src/main/java/com/thalesgroup/hudson/plugins/sourcemonitor/ConfigurableParameters.java

```diff
@@ -0,0 +1,149 @@
+package com.thalesgroup.hudson.plugins.sourcemonitor;
+
+import java.io.Serializable;
+import hudson.FilePath;
+// serializable required for multibranch pipeline
+public class ConfigurableParameters implements Serializable {
+
```

Comment:
It is strongly recommended to declare a serialVersionUID (as per the Serializable interface documentation).